grepper_python/main.py

```diff
@@ -0,0 +1,152 @@
+"""
+The MIT License
+
+Copyright (c) 2010-2023 Grepper, Inc. (https://www.grepper.com)
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+"""
+
+from __future__ import annotations
+
+from typing import Optional
+
+from .answer import GrepperAnswer
+from .exceptions import *
```

Comment:
```suggestion
from grepper_python.answer import GrepperAnswer
from grepper_python.exceptions import BadRequest
from grepper_python.exceptions import Unauthorized
from grepper_python.exceptions import Forbidden
from grepper_python.exceptions import NotFound
from grepper_python.exceptions import MethodNotAllowed
from grepper_python.exceptions import TooManyRequests
from grepper_python.exceptions import InternalServerError
from grepper_python.exceptions import ServiceUnavailable
```

To avoid problems with imports, it's always better to use absolute imports rather than relative imports ;)

Also, you should ALWAYS avoid `import *`

Reply:
Thanks so much for all the reviews!